assets/evaluation_results/squad_v2_gpt-4-0314_question_answering/spec.yaml

```diff
@@ -0,0 +1,33 @@
+type: evaluationresult
+name: squad_v2_gpt-4-0314_question_answering
+version: 2.10.05
+display_name: squad_v2_gpt-4-0314_question_answering
+description: gpt-4-0314 run for squad_v2 dataset
+dataset_family: squad_v2
+dataset_name: squad_v2
+
+model_name: gpt-4-0314
+model_version: "0314"
```

Comment:
model version to be 4 as shown in model's asset registry

Reply:
currently, using the model version for gpt models similar to other datasets in evaluation assets

![image](https://github.com/Azure/azureml-assets/assets/38103118/84cebdd3-49a7-4558-8332-529e51231cb8)